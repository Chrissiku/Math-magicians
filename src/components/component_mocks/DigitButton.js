/* eslint-disable react/prop-types */
/* eslint-disable react/prefer-stateless-function */
/* eslint-disable react/destructuring-assignment */
import React from 'react';

const DigitButton = (props) => (
  <button type="button" onClick={props.e}>
    {props.name}
  </button>
);

export default DigitButton;
