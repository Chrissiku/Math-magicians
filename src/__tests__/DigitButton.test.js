/* eslint-disable react/jsx-props-no-spreading */
import renderer from 'react-test-renderer';
import DigitButton from '../components/component_mocks/DigitButton';

const props = { name: 'Chris', e: jest.fn() };
test('Test Digit Button component', () => {
  const TREE = renderer.create(<DigitButton {...props} />).toJSON();
  expect(TREE).toMatchSnapshot();
});
