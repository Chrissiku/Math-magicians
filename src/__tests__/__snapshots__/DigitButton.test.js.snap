// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Digit Button component 1`] = `
<button
  onClick={[MockFunction]}
  type="button"
>
  Chris
</button>
`;
